main.py

```diff
@@ -1,10 +1,98 @@
+import json
+from pathlib import Path
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
+BASE_DIR = Path(__file__).resolve().parent
+
+
+def create_guild(guild: dict):
+    if guild:
+        guild_name = guild["name"]
+        if not Guild.objects.filter(name=guild_name).exists():
+            if guild["description"]:
+                guild_description = guild["description"]
+                Guild.objects.create(
+                    name=guild_name,
+                    description=guild_description
+                )
+            else:
+                Guild.objects.create(name=guild_name)
+        return guild_name
+    return None
+
+
+def create_race(race_name: str, race_description: str):
+    if not Race.objects.filter(name=race_name).exists():
+        Race.objects.create(
+            name=race_name,
+            description=race_description
+        )
+    return race_name
```

Comment:
better return `Race` object here, not just `race_name`. Then you can use this object and don't make too many queries to the database